README.md

```diff
@@ -2,6 +2,28 @@
 
 This repository's [issue tracker](https://github.com/osrf/infrastructure/issues) is used to receive reports from the Gazebo and ROS community regarding problems with the infrastructure provided by Open Robotics.
 
+Managed projects include
+
+* [Gazebo Answers](https://answers.gazebosim.org)
+* [Gazebo Community](https://community.gazebosim.org)
+* [Gazebo web site](https://gazebosim.org)
+* [Ignition web site](https://ignitionrobotics.org)
+* [OSRF Build farm](https://build.osrfoundation.org)
+* [OSRF package repositories](https://packages.osrfoundation.org)
+* [ROS 2 Build farm repositories](https://repo.ros2.org)
+* [ROS 2 Build farm](https://build.ros2.org)
+* [ROS 2 CI](https://ci.ros2.org)
+* [ROS Answers](https://answers.ros.org)
+* [ROS Build farm repositories](https://repositories.ros.org)
+* [ROS Build farm](https://build.ros.org)
+* [ROS Discourse](https://discourse.ros.org)
+* [ROS bootstrap repositories](https://repos.ros.org)
+* [ROS package repositories](http://packages.ros.org)
+* [ROS snapshot repositories](http://snapshots.ros.org)
+* [ROS static downloads](http://download.ros.org)
+* [ROS web site](https://www.ros.org)
+* [ROS wiki](https://wiki.ros.org)
```

Comment:
I think we should probably add:

* [ROS Documentation](https://docs.ros.org)
* [ROS 2 API documentation](https://docs.ros2.org)
* [ROS package index](https://index.ros.org)

Reply:
Make it so